### ISSUE TITLE
Installation sections doesn't contain relevant installation information.

### ISSUE THREAD

**Issue by StevenDufresne** [WordPress/Documentation-Issue-Tracker#1466]
## Issue Description
The "installation" section is the second most visited (20% of sessions) section of the docs. This is the current list of documentation in that category:

[Hosting WordPress](https://wordpress.org/documentation/article/hosting-wordpress/) (40% of pageviews happen here next)
[Optimization](https://wordpress.org/documentation/article/optimization/) (7% of pageviews happen here next)
[Search Engine Optimization](https://wordpress.org/documentation/article/search-engine-optimization/) (2% of pageviews happen here next)
[Troubleshoot login issues](https://wordpress.org/documentation/article/troubleshoot-login-issues/) (2% of pageviews happen here next)
[Use automated installation](https://wordpress.org/documentation/article/use-automated-installation/)(12% of pageviews happen here next)

Most people click through on "Hosting WordPress" with 30% of those users backtracking immediately.

## URL of the Page with the Issue
[(https://wordpress.org/documentation/category/installation/)](https://wordpress.org/documentation/category/installation/)

## Section of Page with the issue
The whole page.

## Why is this a problem?
The Installation section, states that it will help you install but does not include any content for helping you install WordPress.

| Header | Header |
|--------|--------|
| <img width="1333" alt="Screenshot 2024-02-22 at 1 35 00 PM" src="https://github.com/WordPress/Documentation-Issue-Tracker/assets/1657336/fcb6d4a5-ce7d-4671-9fba-dcfce1e86df8"> | <img width="1329" alt="Screenshot 2024-02-22 at 1 35 28 PM" src="https://github.com/WordPress/Documentation-Issue-Tracker/assets/1657336/e5007096-6dde-432d-a0a6-cc0b698bef9d"> | 

<sub>*Data taken from Jan 1, 2023 to May 1, 2023.</sub>

## Suggested Fix
I think we need to thoughtfully rethink this page. It's really important. As a first step, let's add a link to the page in the first position

"Install WordPress" and link it to https://developer.wordpress.org/advanced-administration/before-install/howto-install/.

Making the page read: 

[Install WordPress](https://developer.wordpress.org/advanced-administration/before-install/howto-install/)
[Hosting WordPress](https://wordpress.org/documentation/article/hosting-wordpress/) 
[Optimization](https://wordpress.org/documentation/article/optimization/)
[Search Engine Optimization](https://wordpress.org/documentation/article/search-engine-optimization/)
[Troubleshoot login issues](https://wordpress.org/documentation/article/troubleshoot-login-issues/)
[Use automated installation](https://wordpress.org/documentation/article/use-automated-installation/)

**Comment by squelchdesign** [comment]
Just my tuppence worth, but could some of those 30% of bounces be people actually looking to _download_ WordPress? Some might also be looking for the [hosting requirements](https://wordpress.org/about/requirements/) page. I'm not understanding why Optimization and SEO is in the installation section either.

**Comment by flexseth** [comment]
Back in the day there was the "Famous Five Minute" install. Quite possibly the first documentation page for WordPress I ever saw... It was one of the pages I remember going back to over and over again when starting to  use WordPress in a larger capacity. From there you could get to the info on permalinks, which linked to Apache settings. 

While I understand it is end-user documentation, what if someone winds up on the documentation page and wants to install WordPress. They end up bouncing and going to a search engine - "how to install wordpress"- find 5 sponsored results from web hosts show up. This could be a limitation to them wanting to _do more_ with WordPress, over time.

As I understand it is important for the Foundation to keep a close relationship with web hosts as they sponsor the project. But also there are people who want to dive deeper and be more hands-on with WordPress. So let's support them too. 

- - - 

Guess the distinction to be made is someone who wants to say 
> I installed WordPress myself!

and a lot of new users who will have a web host, or do the one-click installation through a web host. 

> > > - - -

I agree with @StevenDufresne that it would be good to provide a link to the installation docs.

Looking over documentation pages since they have been migrated, it seems like a lot of content was lost.

---

#### possible solution?
What about providing a simple link at the bottom of the page with a blue callout box: 

> If you would like to install WordPress yourself, please visit [Advanced Administration](https://developer.wordpress.org/advanced-administration/before-install/howto-install/). 

albeit, I will admit that page could use some work too. 

Just thinking about supporting someone who may just want to be a WordPress "end-user"  :)

**Comment by flexseth** [comment]
Scrolled up and accidentally assigned the issue to myself...

Can we have a discussion about the experience, @estelaris?

**Comment by StevenDufresne** [comment]
> We are not adding links to any developer handbooks on documentation. The goal of documentation is to provide information, how to's and guides for non-developers, adding links to developers documentation defeats the purpose.

@estelaris I understand this from a philosophical point of view, but we aren't practicing what we preach. The "optimization" link, the second one in the list, redirects to the developer docs optimization page. How does this differ?

Ultimately none of these articles are relevant to "Installation" and we should either remove the section altogether or provide access to information that relates to its stated objective. I'm having a hard time understanding who this section is serving in its current state and what is justifying keeping it around. The only thing I can see in the data is that people are looking for help installing and being underserved.

> possible solution?
>What about providing a simple link at the bottom of the page:
> If you would like to install WordPress yourself, please visit [Advanced Administration](https://developer.wordpress.org/advanced-administration/before-install/howto-install/).

I would prefer to add it as a first link and monitor anonymous pageview data. I am sensitive to the blurred lines between the docs and the developer docs and would also like to see clearer lines between the two, and if we think doing so causes more problems than it solves, I'm fine with a callout lower down the page for now.

> The Advanced Administrative page needs work too

Agreed. (https://github.com/WordPress/Advanced-administration-handbook/issues/325)

> Just my tuppence worth, but could some of those 30% of bounces be people actually looking to download WordPress?

@squelchdesign It's a possibility. The download page also includes a link to the installation guide so we could also try this as well.

**Comment by estelaris** [comment]
I still don't think that we should add any links to the developer handbooks. These are the solutions I can offer:

1.  rename the articles that should be in HelpHub
2. revise the category description to something like: "Items to be aware before you install WordPress" (this content is under review)
3. remove the articles that have been redirected to the Advanced Admin handbook
4. Add a subtitle or intro paragraph to highlight that HelpHub is "documentation for non-devs"

**Comment by estelaris** [comment]
@flexseth your comment is all about developers. You are not thinking about non-developers. That 5-minute installation was a nightmare to a non-dev and the best that ever happened was the on-click hosting installation.

Developers have numerous resources to figure things out and feel proud of the work they do. Let's leave this section to non-devs only

**Comment by squelchdesign** [comment]
I agree that we shouldn't assume everyone coming to that page is a developer, but what's the harm in beginning with the easy options for non-developers with further reading at the bottom for the more technically minded? These are the _technical_ guides after all.

**Comment by flexseth** [comment]
> @flexseth your comment is all about developers. You are not thinking about non-developers. That 5-minute installation was a nightmare to a non-dev and the best that ever happened was the on-click hosting installation.
> 
> Developers have numerous resources to figure things out and feel proud of the work they do. Let's leave this section to non-devs only

I think the confusion comes (for me) - in that the docs look the same. I didn't look at that as an "end-user" docs page... as I got involved with the documentation process after the new systems were put in place. 

Guess I never really used the "end-user" docs before so maybe that is also a consideration. I'm not looking to discount end-users, but there may be a subset of end-users who want to eventually do more with WordPress. 

So I'm just wondering from a UX perspective, how to see the journey the user takes, over time, to become a more "advanced" user than just an "end-user." I don't know that everyone wants to be a developer, but plenty of people want to better administer their website. That's the overlap / user persona I'm thinking about! :)

**Comment by flexseth** [comment]
Other documentation articles provide waypoints to direct users to the correct place
<img width="804" alt="image" src="https://github.com/WordPress/Documentation-Issue-Tracker/assets/3792502/7495f3b3-5e67-4e78-9cb2-1db606cf13ec">


https://wordpress.org/documentation/article/where-to-find-wordpress-help/

**Comment by flexseth** [comment]
Found a callout when I landed somewhere in support land... maybe a callout to link to more "advanced usage" examples?
<img width="780" alt="Screenshot 2024-04-09 at 1 04 21 PM" src="https://github.com/WordPress/Documentation-Issue-Tracker/assets/3792502/17119fd0-c7b4-4a6c-a78d-4454b4ae4a48">

**Comment by StevenDufresne** [comment]
Thanks @flexseth I saw that above. I feel it is probably too aggressive of a visual for this experiment. It seems to me like it will overshadow the other links and lead to skewed results...?

**Comment by flexseth** [comment]
>  I feel it is probably too aggressive of a visual for this experiment. 

Totally agree, just a sentence with a link to the "self-hosted" install instructions would probably do. I was just pointing out that other docs cross-link to provide guidance where to go :)

**Comment by pooja9712** [comment]
Hii there, I want to do it could you please assign it to me??

**Comment by StevenDufresne** [comment]
HI @pooja9712! Thanks for volunteering. I didn't get a chance to cycle back to this with an example yet. I agree with @flexseth that we can probably add a sentence below the content for now.

I'm thinking something like this:
| Example |
|--------|
| <img width="867" alt="Screenshot 2024-04-16 at 9 56 04 AM" src="https://github.com/WordPress/Documentation-Issue-Tracker/assets/1657336/d4ece45f-3359-4b0f-b997-7939b614faa9"> | 

I'm not sure where this will need to be added, but if it can't be added via documentation files, I can help with any code changes that we may need.

Thanks!
